src/components/content/Dashboard/Dashboard.js

```diff
@@ -89,16 +87,15 @@ class Dashboard extends Component {
 		const displayProjects = projects.map( project => {
 			const index = projects.indexOf(project);
 			return (
-				<li className="create-project-thumb">
-					<Link to={`/user/${userid}/project/${project.id}/ideas`} key={`project-${index}`}>
-						<div className="create-project-thumb-body">
-							<img src="https://cdn1.iconfinder.com/data/icons/creative-concept/174/CREATIVE_CONCEPT_Black-08-512.png" alt="add icon"/>
-						</div>
-						<div className="create-project-thumb-footer">
-							<label>{project.name}</label>
-						</div>
-					</Link>
-				</li>
+				<Link to={`/user/${userid}/project/${project.id}/ideas`} key={`project-${index}`}>
```

Comment:
Here, the "Link" tag is the parent of the "list" tag, earlier it's the opposite, which is it?

Reply:
The "Link" tag should be placed around the whole "project thumb" or "create project thumb" it doesn't make the user click on specific areas of the thumb to get to their project.